poetry/console/commands/version.py

```diff
@@ -55,7 +55,7 @@ def handle(self) -> None:
             )
 
             if self.option("short"):
-                self.line("{}".format(version))
+                self.line(f"{version}")
```

Comment:
this can be simpler I guess?
```suggestion
                self.line(version)
```

Reply:
Might want to use `str(version)` or `version.to_string()`

Reply:
went with `str(version)` , thank you 👍